### DISCUSSION TITLE
How to show tiff file in Polygon component?

### DISCUSSION THREAD

**Discussion by pavanbora-tudip** [react-native-maps/react-native-maps#4276] [Q&A]
We have to show the tiff file inside the polygon component of react-native-maps. But we are unable to add images/tiff files inside the polygon component. Is there any possible way to add images/tiff files in react-native-maps?

![Screenshot (24)](https://user-images.githubusercontent.com/83057677/173865947-1ba3b5ed-e3e0-4a2a-b023-ede2d4e1e512.png)

**Comment by Willham12** [comment]
Not sure if possible, see https://github.com/react-native-maps/react-native-maps/blob/master/docs/overlay.md

alternative you can use mapbox, see https://github.com/nitaliano/react-native-mapbox-gl/blob/master/docs/ImageSource.md
